core/src/main/java/org/mskcc/oncotree/crosswalk/MSKConcept.java

```diff
@@ -81,6 +82,30 @@ public void setCrosswalks(HashMap<String, List<String>> crosswalks) {
         this.crosswalks = crosswalks;
     }
 
+    /**
+    *
+    * @return history
+    */
+    public Set<String> getHistory() {
+        return history;
+    }
+
+    /**
+    *
+    * @param oncotreeCode
+    */
+    public void addHistory(String oncotreeCode) {
```

Comment:
A little unusual to have a get() / add() combination, but it seems to work here. To be truly multi-purpose, a set() method could be provided which overwrote the set, but we don't need it now.